client/src/components/brushableHistogram/index.js

```diff
@@ -87,23 +86,42 @@ class HistogramBrush extends React.Component {
   componentDidUpdate(prevProps) {
     const { field, world, continuousSelection } = this.props;
     const { x, y, bins, svgRef } = this._histogram;
+    let { brushXselection, brushX } = this.state;
+    let forceBrushUpdate = false;
 
-    if (world !== prevProps.world) {
-      this.renderAxesBrushBins(x, y, bins, svgRef, field);
+    /*
+    Update our axis if the underlying dataframe column has changed
+    */
```

Comment:
Does this mean that if a column is unchanged it keeps it's selection after clipping? If so does this work with data that has a high number of cells with the same value for one continuous metadata label - ie does it cause the brush to stay if a user clips the data but that category is not clipped?

Reply:
Columns are always changed if they are continuous and the clip values actually change (ie, immutable data structure, so we never change an existing column in place).

Essentially this code reflects the logic where:
* we update the histogram bins and scale whenever the column changes
* so we need to update the SVG labels and brush whenever the column changes

Reply:
Got it! I was confusing this with how the zero removal from histograms was handled so that you can sometimes clip but still have the same histogram.